data/json/items/resources/alien.json

```
@@ -145,5 +145,96 @@
     "price": 2000,
     "material": [ "copper", "ceramic" ],
     "category": "spare_parts"
+  },
+  {
+    "id": "exodii_ingot_aluminum",
+    "category": "spare_parts",
+    "name": { "str": "Enormous aluminum ingot" },
```

```suggestion
    "type": "GENERIC",
    "id": "exodii_ingot_aluminum",
    "category": "spare_parts",
    "name": { "str": "enormous aluminum ingot" },
```
Capitalization & type applies to all ingots